controls/radtimespanpicker/timespanpicker-key-properties.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Key Properties
+page_title: Key Properties
+description: Key Properties
+slug: radtimespanpicker-features-key-properties
+tags: properties,features,key,properties
+published: True
+position: 3
+---
+
+# Key Properties
+
+The purpose of this help article is to provide detailed information about some of the key properties of the __RadTimeSpanPicker__ control.
+<Comment: Is this article going to get longer soon? If not, the sentence above seems odd to me. It claims that the article will discuss key properties (plural) of the control but only one property is discussed. Perhaps reword the previous sentence as "This help article will provide information about the __StringFormat__ property, a key property of the __RadTimeSpanPicker__ control." Later, when you have a few properties to discuss, the sentence above seems OK. ALSO, the header 1 "Key Properties" seems wrong since there is only one property. I suggest you change it to "StrongFormat Property" until this article expands to discuss more than one property. The same goes for the title, page_title and description above. They all suggest more than one property in this article will be discussed.>
```

Comment:
Currently the control has just one specific property the "StringFormat". There wasn't an appropriate article where we could put the information about it, so we decided to create a "Key Properties" article (many of our other controls has such article). I will preserve the name of the article as our clients will look for such article when they are searching for information about the specific properties of the TimeSpanPeaker even though currently it is just one,  but will change the first sentence as suggested. Sooner or later new specific properties will be created for the control and then we will change the first sentence. Thanks for the feedback. :+1: